crates/rethnet_eth/Cargo.toml

```diff
@@ -20,7 +20,7 @@ reqwest = { version = "0.11", features = ["blocking", "json"] }
 # https://github.com/TrueLayer/reqwest-middleware/blob/main/reqwest-retry/CHANGELOG.md#unreleased
 reqwest-middleware = { git = "https://github.com/TrueLayer/reqwest-middleware", rev = "a54319a", default-features = false }
 reqwest-retry = { git = "https://github.com/TrueLayer/reqwest-middleware", rev = "a54319a", default-features = false }
-revm-primitives = { git = "https://github.com/Wodann/revm", rev = "b6f8ebf", version = "1.1", default-features = false }
+revm-primitives = { git = "https://github.com/agostbiro/revm", rev = "430502e", version = "1.1", default-features = false }
```

Comment:
will need to change to Wodann/revm once the [PR](https://github.com/Wodann/revm/pull/2) is merged